hudi-client/hudi-java-client/pom.xml

```diff
@@ -42,6 +42,11 @@
             <artifactId>hudi-client-common</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.hudi</groupId>
+            <artifactId>hudi-hadoop-common</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
Move this above `hudi-client-common`.  Also, I think `hudi-client-common` pulls in `hudi-hadoop-common`?  Similar for other POMs.

Reply:
Same thing I said above. We can try removing it when the rest of the review is done. But don't want to mess around with this too much while we are making other changes

Reply:
Sg.